### PR TITLE
enhancement: Add LEP for Prometheus support

### PR DESCRIPTION
**Pull request by PhanLe1010**
Issues:

longhorn/longhorn#1180
longhorn/longhorn#1832

### CODE REVIEW

**Comment by yasker** [comment]
Any update?

**Comment by joshimoo** [comment]
@PhanLe1010 I think this requires an update, to be consistent with the implementation?

**Comment by yasker** [comment]
Any update?

**Comment by PhanLe1010** [comment]
@joshimoo @yasker 

I will update and finish this and https://github.com/longhorn/longhorn/issues/1832 today.

File: enhancements/20200909-prometheus-support.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+
+See the [User Experience In Detail](#user-experience-in-detail) section for definition of each metric.
+
+### Non-goals
+
+We are not planing to expose 6 metrics in this release:
+1. longhorn_backup_stats_number_failed_backups 
+1. longhorn_backup_stats_number_succeed_backups
+1. longhorn_backup_stats_backup_status (status for this backup (0=InProgress,1=Done,2=Failed))
+1. longhorn_volume_iops
```
[the remark this answers is not in the document]
Comment (PhanLe1010) [reply enhancements/20200909-prometheus-support.md]: Thank you! Fixed

**Review by yasker — APPROVED** [review]

**Comment by yasker** [comment]
LGTM. @joshimoo ?

**Comment by yasker** [comment]
@joshimoo is on vacation. I will merge this for now.

**Pull request merged**